packages/terra-horizontal-divider/tests/jest/HorizontalDivider.test.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import HorizontalDivider from '../../src/HorizontalDivider';
+
+describe('HorizontalDivider', () => {
+  const defaultRender = <HorizontalDivider />;
+
+  // Snapshot Tests
+  it('should render a basic component', () => {
+    const wrapper = shallow(defaultRender);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  // Structure Tests
+  it('should have the class divider', () => {
```

Comment:
This is tested in the first test by comparing the snapshots. You don't need both tests.

Reply:
Updated https://github.com/cerner/terra-core/pull/962/commits/5fb27563217e211a5ec708f6dc65da66fb369f31#diff-46429ad4c25a0b00f25c0054ce0dddf6L12